src/parser.js

```diff
@@ -73,7 +73,13 @@ const parser = yargs
   })
   .coerce('prettier', config => {
     if (typeof config === 'object') {
-      return camelcaseKeys(config)
+      const cfg = camelcaseKeys(config)
+      return Object.keys(cfg).reduce(
+        key =>
+          cfg[key] =
+            cfg[key] === 'true' || (cfg[key] === 'false' ? false : cfg[key]),
+        {},
+      )
```

Comment:
Making `case-insensitive` would be pretty easy. Being a simple 1-line I didn't feel the need to introduce an extra dependency.

Both approaches works fine for me, I suppose we should wait @kentcdodds opinion on that matter.